MODULE.bazel.lock

```diff
@@ -1123,6 +1123,27 @@
         "recordedRepoMappingEntries": []
       }
     },
+    "@@bazel_tools//tools/test:extensions.bzl%remote_coverage_tools_extension": {
```

Comment:
This keeps popping up in all of my branches due to using [bazel-watcher](https://github.com/bazelbuild/bazel-watcher).

It's OK to include here